udata/core/reuse/models.py

```diff
@@ -40,12 +41,14 @@ def hidden(self):
                     db.Q(datasets__0__exists=False) |
                     db.Q(deleted__ne=None))
 
-
+decorator = mdstrip_field('description')
+@decorator
```

Comment:
I guess that's because `apply` parameters are wrong in this case and thus the signal is not sent.